articles/quickstart.md

```diff
@@ -1,163 +1,129 @@
-If you are using a different development environment, like Visual Studio Code, please 
-follow the instructions to call the .NET Core SDK from the command line.
+1. Create a file called `host.py`. This file will contain your python host code.
 
-### Step 1: Create a Project and Solution
+#### [C# Command Line](#tab/tabid-csharp)
 
-#### [Command Line / Visual Studio Code](#tab/tabid-vscode)
+1. Create a new Q# project:
 
-Run the following commands in your favorite command line (e.g.: PowerShell or Bash):
+    ```bash
+    dotnet new console -lang Q# --output Bell
+    cd Bell
+    ```
 
-```bash
-$ dotnet new console -lang Q# --output Bell
-$ cd Bell 
-$ code . # To open in Visual Studio Code.
-```
+    You should see a `.csproj` file, a Q# file called `Operation.qs`, and a host program file called `Driver.cs`
+
+1. Rename the Q# file
+
+    ```bash
+    mv Operation.qs Bell.qs
+    ```
 
 #### [Visual Studio](#tab/tabid-vs2019)
 
-- Open up Visual Studio.
-- Go to the `File` menu and select `New` > `Project...`.
-- In the project template explorer, under `Installed` > `Visual C#`, select the `Q# Application` template.
-- Make sure you have `.NET Framework 4.6.1` selected in the list at the bottom of the `New Project` dialog box.
-- Give your project the name `Bell`.
+1. Create a new project
 
-***
+   * Open Visual Studio
+   * Go to the **File** menu and select **New** -> **Project...**
+   * In the project template explorer, under `Installed` > `Visual C#`, select the `Q# Application` template
+   * Make sure you have `.NET Framework 4.6.1` selected in the list at the bottom of the `New Project` dialog box.
+   * Give your project the name `Bell`
 
-### Step 2: Enter the Q# Code
+1. Rename the Q# file
 
-Our goal is to create a [Bell State](https://en.wikipedia.org/wiki/Bell_state) showing entanglement. We will build this up piece by piece to show the concepts of qubit state, gates and measurement.
+   * Navigate to the **Solution Explorer**
+   * Right click on the `Operation.qs` file
+   * Rename it to `Bell.qs`
 
-#### Source files for a quantum application
-Your development environment should have two files open:
-`Driver.cs`, which will hold the C# driver for your quantum code,
-and `Operations.qs`, which will hold the quantum code itself.
+* * *
 
-#### Q# source
-The first step is to rename the Q# file to `Bell.qs`.
-Right-click on `Operations.qs` in the Visual Studio Solution Explorer (Ctrl+Alt+L to focus) or the Visual Studio Code Explorer (Ctrl/⌘+Shift+E to focus), and select the Rename option.
-Replace `Operations` with `Bell` and hit return.
+## Write a Q# operation
 
-To enter the Q# code, make sure that you are editing the
-`Bell.qs` window.
-The `Bell.qs` file should have the following contents:
+Our goal is to create a [Bell State](https://en.wikipedia.org/wiki/Bell_state) showing entanglement. We will build this up piece by piece to show the concepts of qubit state, gates, and measurement.
```

Comment:
```suggestion
Our goal is to create a [Bell state](https://en.wikipedia.org/wiki/Bell_state) showing entanglement. We will build this up piece by piece to show the concepts of qubit state, gates, and measurement.
```

Reply:
Unfortunately trying to get rid of the awkwardness of the "concept of qubit state," I added awkward alliteration. Maybe just "...of a qubit state, ..."?

Reply:
maybe "...to demonstrate qubit states, gates, and measurement."